### PR TITLE
Create footer component in container and bump in new psammead styling

### LINKED ISSUE #1453: Footer should have 56px padding to distance itself from content
**Is your feature request related to a problem? Please describe.**
Designs have a 56px gap between Footer and the last paragraph of the page.

**Describe the solution you'd like**
Update [the Footer container](https://github.com/bbc/simorgh/blob/latest/src/app/containers/Footer/index.jsx) to have 56px of space between the footer and the content above it.

PR should have screenshots showing before & after.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Snapshot tests need updating.

**Additional context**
N/a

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #1453

**Overall change:** _Creates a styled.footer with 56px margin-top. Also pulls in the new versions of psammead-figure, psammead-headings and psammead-paragraph which contain spacing updates._

**Code changes:**

- _Creates `styled.footer` in the container with 7rem (56px) margin-top_
- _Pulls in the new styled for psammead-figure, psammead-headings and psammead-paragraph_

**NB: I've purposefully created the Footer styled component in the container so we can get this out for launch. There is an [issue in psammead to create psammead-footer](https://github.com/bbc/psammead/issues/395) and a follow up [issue in simorgh to use the newly created psammead-footer](https://github.com/bbc/simorgh/issues/1492).**

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [x] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by dr3 — DISMISSED** [review]
LGTM on the basis you've made issues to extract it.

Forward refs are a shame but will be resolved by moving to psammead or using `react testing library`. As this is the intention, and this is tiny, Im happy :)

File: src/app/containers/Footer/index.jsx [hunk 1]
```diff
@@ -1,7 +1,13 @@
+import styled from 'styled-components';
 import React from 'react';
 import SitewideLinks from '@bbc/psammead-sitewide-links';
 import BrandContainer from '../Brand';
 
+// 3.5rem = 56px = 7 GEL_SPACING's
```
Comment (ChrisBAshton) [inline src/app/containers/Footer/index.jsx]: Why are we not using `GEL_SPACING` directly? Just curious - no need to change.

```suggestion
// 3.5rem = 56px = 7 x GEL_SPACING
```

**Review by ChrisBAshton — COMMENTED** [review]
A quick question and a comment tweak 👍 looks good

**Review by dr3 — DISMISSED** [review]

**Review by sareh — DISMISSED** [review]

**Review by sareh — APPROVED** [review]

**Comment by pjlee11** [comment]
## 375px viewport
v1-spacing-375

## 600px viewport
v1-spacing-600

## 1008px viewport
v1-spacing-1008

## 1280px viewport
v1-spacing-1280

**Review by dr3 — APPROVED** [review]

**Pull request merged**